db/versionHistory.go

```diff
@@ -72,5 +72,6 @@ func init() {
 		{Major: 2, Minor: 3, Patch: 2},
 		{Major: 2, Minor: 4},
 		{Major: 2, Minor: 5},
+		{Major: 2, Minor: 6},
```

Comment:
please merge this with the existing 2.5 migration as they will come out in the same release

Reply:
Can you be more specific as to what other PRs I should merge? At the moment, it looks only like #463, as #457 does not contain any go code. #396 is merged.

Reply:
Version should be set to 2.5 here and you should merge your sql migration with the v2.5.0.sql file which is already in the development branch migrations folder.